hs/src/Reach/Simulator/Core.hs

```diff
@@ -199,6 +177,7 @@ data Action
   | A_ChangeActor Int
   | A_AdvanceTime Integer
   | A_AdvanceSeconds Integer
+  | A_InteractV String String DLType
```

Comment:
There should be some protection that these are always the same value. I imagined that when you created a participant, you would have to specify all of the interact values up front